src/appshell/qml/HomePage/HomePage.qml

```diff
@@ -125,7 +125,9 @@ DockPage {
     Component {
         id: learnComp
 
-        LearnPage {}
+        LearnPage {
+            item: root.subItem
```

Comment:
I think we need to think about more understandable names, otherwise, it’s not clear what it is

Reply:
initiallySectionName?

musescore://home?initiallySectionName=add-ons&initiallySubsectionName=languages

Reply:
maybe just `section` 

musescore://home?section=add-ons&section=languages

Reply:
section and subSection
they should be different because it's just assigning a value to a property when creating a component